### PR TITLE
Make golangcross-build images standalone

### PR DESCRIPTION
**Pull request by mikemadden42**
- Rename main to amd64

### CODE REVIEW

**Comment by mikemadden42** [comment]
Previous work was started in this PR, but I quickly ran into multiarch issues.  The current PR should solve the multiarch issues eventually.

https://github.com/elastic/golang-crossbuild/pull/52

**Comment by mikemadden42** [comment]
Here's is the current list of images that are produced for this PR.  Let me know your thoughts on the names.  I need to refine the list of underlying packages for amd64 & i386.

```
$ docker image ls -a | grep golang-crossbuild | sort -k2
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-amd64           892eeb8ae79a        6 minutes ago       1.68GB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-amd64-debian7   7bba53b09712        6 minutes ago       1GB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-amd64-debian8   5ade60873475        6 minutes ago       1.38GB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-arm             d889a3633be5        6 minutes ago       1.08GB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-base            e3826c2a90b9        46 hours ago        698MB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-base-debian7    36d13e692f52        46 hours ago        577MB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-base-debian8    e2614790747d        46 hours ago        672MB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-darwin          e34d3899f6a3        6 minutes ago       1.27GB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-i386            bb6a147ecf6e        6 minutes ago       1.68GB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-i386-debian7    bbc7284fa50d        6 minutes ago       1GB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-i386-debian8    061df841b1d7        6 minutes ago       1.38GB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-main            7e6eec888c4c        46 hours ago        1.68GB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-main-debian7    4b435bd0ad5f        46 hours ago        1GB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-main-debian8    ecc59362b45a        46 hours ago        1.38GB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-mips            1a5111bbaeb4        6 minutes ago       1.18GB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-ppc             991d519d6459        6 minutes ago       988MB
docker.elastic.co/beats-dev/golang-crossbuild   1.13.12-s390x           edf4c5537a44        6 minutes ago       800MB
```

File: go1.13/amd64/Dockerfile.tmpl [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        patch \
+        xz-utils \
+    && rm -rf /var/lib/apt/lists/*
+
+{{if (eq .DEBIAN_VERSION "7") -}}
+RUN \
+    apt-get -o Acquire::Check-Valid-Until=false update \
+    && apt-get install -y --no-install-recommends --allow-unauthenticated libsystemd-journal-dev \
+    && rm -rf /var/lib/apt/lists/*
+{{else if or (eq .DEBIAN_VERSION "8") (eq .DEBIAN_VERSION "9") -}}
```
Comment (mdelapenya) [inline go1.13/amd64/Dockerfile.tmpl:27]: Maybe a `greaterThan` here?

File: go1.13/i386/Dockerfile.tmpl [hunk 2]
```diff
@@ -13,13 +14,26 @@ RUN \
...
         patch \
         xz-utils \
     && rm -rf /var/lib/apt/lists/*
 
+{{if (eq .DEBIAN_VERSION "7") -}}
+RUN \
+    apt-get -o Acquire::Check-Valid-Until=false update \
+    && apt-get install -y --no-install-recommends --allow-unauthenticated libsystemd-journal-dev:i386 \
+    && rm -rf /var/lib/apt/lists/*
+{{else if or (eq .DEBIAN_VERSION "8") (eq .DEBIAN_VERSION "9") -}}
```
Comment (mdelapenya) [inline go1.13/i386/Dockerfile.tmpl:30]: Same here

**Pull request closed without merging**